pkgs/applications/science/logic/potassco/clingcon.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, cmake, bison, re2c }:
```

To my personal taste (you don't have to oblige with this) I like putting each "import" in it's own line:

```suggestion
{ stdenv
, fetchFromGitHub
, cmake
, bison
, re2c
}:
```

This way diffs are prettier and clearer IMO.